packages/core/src/utils/components.ts

```diff
@@ -41,3 +44,11 @@ export const isComponentAllowedOnRoot = ({ type, category, componentId }: Compon
 export const isStructureWithRelativeHeight = (componentId?: string, height?: string | number) => {
   return isContentfulStructureComponent(componentId) && !height?.toString().endsWith('px');
 };
+
+const stylesToKeep = ['cfImageAsset'];
+const stylesToRemove = CF_STYLE_ATTRIBUTES.filter((style) => !stylesToKeep.includes(style));
+const propsToRemove = ['cfHyperlink', 'cfOpenInNewTab', 'cfSsrClassName'];
+
+export const sanitizeNodeProps = (nodeProps: Record<PropertyKey, PrimitiveValue>) => {
```

Comment:
As I found out when cleaning up the deps for NumericUnitInput, continously adding util functions to a file can cause dep havoc. Any component that needs to import `sanitizeNodeProps` will have to import everything in this file, and if theres some deps in here with dependency violations, any subsequent components could unknowingly inherit those violations.

Therefore, its probably safer to put each util into their own file

Reply:
Also just realized this is the experiences repo, not UI, so probably not that big of a deal here, but could be in the future.

Reply:
Good point. I'll move this to its own file.